_episodes/02-filedir.md

```diff
@@ -101,7 +101,7 @@ because its name begins with `/`.
 > Notice that there are two meanings for the `/` character.
 > When it appears at the front of a file or directory name,
 > it refers to the root directory. When it appears *inside* a name,
-> it's just a separator.
```

Comment:
I not sure this removal is appropriate.

Reply:
Yes there were a few I wasn't sure about. I'll fix this one.